<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelWindow</class>
 <widget class="QMainWindow" name="LevelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>575</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Level</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="10" colspan="2">
     <widget class="QPushButton" name="Btn20">
      <property name="text">
       <string>t</string>
      </property>
     </widget>
    </item>
    <item row="3" column="22" colspan="2">
     <widget class="QPushButton" name="Btn13">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="21" colspan="2">
     <widget class="QPushButton" name="Btn52">
      <property name="text">
       <string>/    ?</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="26">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	font-size: 20px;
}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="24" colspan="2">
     <widget class="QPushButton" name="Btn14">
      <property name="text">
       <string>BackSpace</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="Btn15">
      <property name="text">
       <string>Tab</string>
      </property>
     </widget>
    </item>
    <item row="5" column="14" colspan="2">
     <widget class="QPushButton" name="Btn36">
      <property name="text">
       <string>j</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
      <property name="autoRepeatInterval">
       <number>100</number>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QPushButton" name="Btn43">
      <property name="text">
       <string>z</string>
      </property>
     </widget>
    </item>
    <item row="4" column="12" colspan="2">
     <widget class="QPushButton" name="Btn21">
      <property name="text">
       <string>y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="20" colspan="2">
     <widget class="QPushButton" name="Btn39">
      <property name="text">
       <string>;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QPushButton" name="Btn31">
      <property name="text">
       <string>s</string>
      </property>
     </widget>
    </item>
    <item row="7" column="24" colspan="2">
     <widget class="QPushButton" name="Btn60">
      <property name="text">
       <string>Ctrl</string>
      </property>
     </widget>
    </item>
    <item row="6" column="17" colspan="2">
     <widget class="QPushButton" name="Btn50">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="Btn54">
      <property name="text">
       <string>Ctrl</string>
      </property>
     </widget>
    </item>
    <item row="4" column="20" colspan="2">
     <widget class="QPushButton" name="Btn25">
      <property name="text">
       <string>p</string>
      </property>
     </widget>
    </item>
    <item row="3" column="14" colspan="2">
     <widget class="QPushButton" name="Btn9">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="26">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	font-size: 20px;
}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1282</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="5" column="16" colspan="2">
     <widget class="QPushButton" name="Btn37">
      <property name="text">
       <string>k</string>
      </property>
     </widget>
    </item>
    <item row="5" column="24" colspan="2">
     <widget class="QPushButton" name="Btn41">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="4">
     <widget class="QPushButton" name="Btn56">
      <property name="text">
       <string>Alt</string>
      </property>
     </widget>
    </item>
    <item row="4" column="24">
     <widget class="QPushButton" name="Btn27">
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="2">
     <widget class="QPushButton" name="Btn44">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="23" colspan="3">
     <widget class="QPushButton" name="Btn53">
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
    </item>
    <item row="6" column="15" colspan="2">
     <widget class="QPushButton" name="Btn49">
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QPushButton" name="Btn16">
      <property name="text">
       <string>q</string>
      </property>
     </widget>
    </item>
    <item row="4" column="25">
     <widget class="QPushButton" name="Btn28">
      <property name="text">
       <string>\</string>
      </property>
     </widget>
    </item>
    <item row="7" column="18" colspan="4">
     <widget class="QPushButton" name="Btn58">
      <property name="text">
       <string>Alt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="Btn3">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9" colspan="2">
     <widget class="QPushButton" name="Btn46">
      <property name="text">
       <string>v</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8" colspan="2">
     <widget class="QPushButton" name="Btn19">
      <property name="text">
       <string>r</string>
      </property>
     </widget>
    </item>
    <item row="3" column="20" colspan="2">
     <widget class="QPushButton" name="Btn12">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="Btn42">
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QPushButton" name="Btn4">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="18" colspan="2">
     <widget class="QPushButton" name="Btn11">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="18" colspan="2">
     <widget class="QPushButton" name="Btn24">
      <property name="text">
       <string>o</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6" colspan="2">
     <widget class="QPushButton" name="Btn18">
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="4" column="14" colspan="2">
     <widget class="QPushButton" name="Btn22">
      <property name="text">
       <string>u</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8" colspan="2">
     <widget class="QPushButton" name="Btn6">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="6" column="13" colspan="2">
     <widget class="QPushButton" name="Btn48">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="Btn29">
      <property name="text">
       <string>CapsLock</string>
      </property>
     </widget>
    </item>
    <item row="4" column="22" colspan="2">
     <widget class="QPushButton" name="Btn26">
      <property name="text">
       <string>[</string>
      </property>
     </widget>
    </item>
    <item row="6" column="19" colspan="2">
     <widget class="QPushButton" name="Btn51">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8" colspan="2">
     <widget class="QPushButton" name="Btn33">
      <property name="text">
       <string>f</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6" colspan="2">
     <widget class="QPushButton" name="Btn5">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="7" column="22" colspan="2">
     <widget class="QPushButton" name="Btn59">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QPushButton" name="Btn17">
      <property name="text">
       <string>w</string>
      </property>
     </widget>
    </item>
    <item row="5" column="10" colspan="2">
     <widget class="QPushButton" name="Btn34">
      <property name="text">
       <string>g</string>
      </property>
     </widget>
    </item>
    <item row="4" column="16" colspan="2">
     <widget class="QPushButton" name="Btn23">
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="5" column="12" colspan="2">
     <widget class="QPushButton" name="Btn35">
      <property name="text">
       <string>h</string>
      </property>
     </widget>
    </item>
    <item row="6" column="11" colspan="2">
     <widget class="QPushButton" name="Btn47">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
    <item row="3" column="12" colspan="2">
     <widget class="QPushButton" name="Btn8">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="18" colspan="2">
     <widget class="QPushButton" name="Btn38">
      <property name="text">
       <string>l</string>
      </property>
     </widget>
    </item>
    <item row="7" column="8" colspan="10">
     <widget class="QPushButton" name="Btn57">
      <property name="text">
       <string>Space</string>
      </property>
     </widget>
    </item>
    <item row="5" column="22" colspan="2">
     <widget class="QPushButton" name="Btn40">
      <property name="text">
       <string>'</string>
      </property>
     </widget>
    </item>
    <item row="3" column="16" colspan="2">
     <widget class="QPushButton" name="Btn10">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6" colspan="2">
     <widget class="QPushButton" name="Btn32">
      <property name="text">
       <string>d</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Btn2">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Btn1">
      <property name="text">
       <string>`</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7" colspan="2">
     <widget class="QPushButton" name="Btn45">
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10" colspan="2">
     <widget class="QPushButton" name="Btn7">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QPushButton" name="Btn30">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QPushButton" name="Btn55">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>Btn9</zorder>
   <zorder>textEdit</zorder>
   <zorder>Btn43</zorder>
   <zorder>Btn35</zorder>
   <zorder>Btn50</zorder>
   <zorder>Btn18</zorder>
   <zorder>Btn25</zorder>
   <zorder>Btn17</zorder>
   <zorder>Btn16</zorder>
   <zorder>Btn33</zorder>
   <zorder>Btn31</zorder>
   <zorder>Btn34</zorder>
   <zorder>Btn49</zorder>
   <zorder>Btn22</zorder>
   <zorder>Btn44</zorder>
   <zorder>Btn27</zorder>
   <zorder>Btn46</zorder>
   <zorder>Btn51</zorder>
   <zorder>Btn24</zorder>
   <zorder>Btn20</zorder>
   <zorder>Btn48</zorder>
   <zorder>Btn21</zorder>
   <zorder>Btn40</zorder>
   <zorder>Btn47</zorder>
   <zorder>Btn37</zorder>
   <zorder>Btn39</zorder>
   <zorder>Btn26</zorder>
   <zorder>Btn45</zorder>
   <zorder>Btn38</zorder>
   <zorder>Btn52</zorder>
   <zorder>Btn32</zorder>
   <zorder>Btn23</zorder>
   <zorder>Btn36</zorder>
   <zorder>Btn30</zorder>
   <zorder>Btn19</zorder>
   <zorder>Btn15</zorder>
   <zorder>Btn28</zorder>
   <zorder>Btn29</zorder>
   <zorder>Btn41</zorder>
   <zorder>Btn42</zorder>
   <zorder>Btn14</zorder>
   <zorder>Btn10</zorder>
   <zorder>Btn12</zorder>
   <zorder>Btn8</zorder>
   <zorder>Btn11</zorder>
   <zorder>Btn13</zorder>
   <zorder>Btn7</zorder>
   <zorder>Btn4</zorder>
   <zorder>Btn5</zorder>
   <zorder>Btn3</zorder>
   <zorder>Btn6</zorder>
   <zorder>Btn1</zorder>
   <zorder>Btn2</zorder>
   <zorder>Btn53</zorder>
   <zorder>Btn54</zorder>
   <zorder>Btn56</zorder>
   <zorder>Btn60</zorder>
   <zorder>Btn58</zorder>
   <zorder>Btn57</zorder>
   <zorder>Btn55</zorder>
   <zorder>Btn59</zorder>
   <zorder>scrollArea</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
